standards/peer-review.md

```diff
@@ -1,8 +1,30 @@
+## Pull Request Workflow
+
+All Pull Requests to NYPL repositories SHOULD be reviewed prior to merging.
```

Comment:
Why not MUST?

Reply:
Yeah, I'll pull that up and merge these two points. My thinking was that there are at times small teams where reviews are more difficult to come by and while our main effort should be to make that not happen, also this helps prevent blockage? Maybe unreviewed merging should be allowed only if there is a written and approved policy?

Reply:
@hokei Updated this policy to include the requirement for a written policy. Also wondering if I should add a policy that if the requester specifically asks for permission to merge without a review and is granted? I know I have been asked for this because the engineer clearly could explain why the code was safe to merge without a review and I couldn't see any capacity to complete the review quickly.